Scheme.md

```diff
@@ -0,0 +1,88 @@
+A key difference between Isolated Web Apps and normal web pages is that Isolated Web Apps do not rely on DNS name resolution and HTTPS certificate authorities.
+Instead, they need to be downloaded and installed from an app store or be distributed via enterprise configuration.
+A user agent can verify the integrity of an Isolated Web App by checking the signature and comparing its corresponding public key to a list of known trusted public keys.
+
+## `isolated-app:` Scheme
+
+Since Isolated Web Apps do not use HTTPS certificate authorities and also need to be independent of domains and name resolution, a new URL scheme is needed to signify this difference.
+The tentative name of this new scheme is `isolated-app:`, highlighting how Isolated Web Apps are isolated from each other and the web through enforced Content Security Policy and Cross-Origin Isolation.
+When loading an `isolated-app:` URL, the user agent checks the signatures of the underlying web bundle and whether it trusts the public key that was used to sign it.
+A user agent might trust, for example, public keys configured via enterprise policy, public keys of known distribution mechanisms/stores, or separately allow-listed public keys of individual Isolated Web Apps.
+`isolated-app:` URLs are considered Secure Contexts, just like HTTPS pages, and thus have access to APIs like Service Workers. `isolated-app:` URLs look like so:
+
+```
+isolated-app://signed-web-bundle-id/path/inside/app.js?some-query#foo
+^           ://^                   /^                 ?^         #^
+scheme         opaque host          path               query      fragment
+```
+
+### Host
+
+URLs with the `isolated-app:` scheme use a Signed Web Bundle ID (see next section) as their opaque host, as defined in the [URL standard](https://url.spec.whatwg.org/).
+The Signed Web Bundle ID being the host adds the requirement that it must be a valid hostname (see [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986)), which is the case given that Signed Web Bundle IDs are [base32-encoded](https://datatracker.ietf.org/doc/html/rfc4648) without padding.
+It is the user agent’s responsibility to map the Signed Web Bundle ID contained in the opaque host to an installed Isolated Web App and then use the URL path to serve resources from its Web Bundle.
+
+### Port & Credentials
+
+Isolated Web App URLs do not use usernames, passwords, or ports.
+All three of these components are therefore always `null`.
+Isolated Web App URLs with a port or credentials should not be loaded and result in an HTTP error.
+
+### Path, Query, Fragment
+
+These work like one would expect from HTTP(S) URLs, as described in the URL specification.
+
+## Signed Web Bundle IDs
+
+Isolated Web Apps are identified by their _Signed Web Bundle ID_.
+Signed Web Bundle IDs consist of an identifier followed by a suffix that represents the type of identifier.
+Both parts are concatenated and base32-encoded, discarding any potential padding.
```

Comment:
It may be worth specifying that to decode a Signed Web Bundle ID the padding can be restored by adding padding characters until the input length is a multiple of 8.

Reply:
Done, PTAL.